<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccItemWidget</class>
 <widget class="QWidget" name="AccItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AccItemWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AccItemWidget{background-color: rgb(255, 255, 255);};</string>
  </property>
  <widget class="QWidget" name="FilletWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>730</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>730</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>730</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#FilletWidget{background:rgb(252,252,252);border-radius:4px;border:1px solid rgb(235,235,235);}
QWidget#FilletWidget:hover,pressed{background:rgb(246,251,255);border-radius:4px;border:1px solid rgb(187,223,255);};</string>
   </property>
   <widget class="QLabel" name="labelAvatar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>10</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:rgb(51,51,51);
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTaxNo">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>34</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:rgb(102,102,102);
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>647</x>
      <y>0</y>
      <width>82</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-top-left-radius:0px;border-bottom-left-radius:0px;background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-top-right-radius:4px;border-bottom-right-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-top-right-radius:4px;border-bottom-right-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
   <widget class="QToolButton" name="loginedButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>82</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:4px;border-bottom-right-radius:4px;font: 14px &quot;微软雅黑&quot;;color:#1E8BED;padding-left:5px;
background-color: rgba(255, 255, 255, 0);border:0px;</string>
    </property>
    <property name="text">
     <string>已登录</string>
    </property>
    <property name="icon">
     <iconset resource="EntCenterWidget.qrc">
      <normaloff>:/EntCenterWidget/image/notExpired.png</normaloff>:/EntCenterWidget/image/notExpired.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>14</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QWidget" name="inReviewWidget" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>82</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>60</height>
     </size>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 17px &quot;微软雅黑&quot;;
color:rgba(244,114,63,1);
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
     </property>
     <property name="text">
      <string>审核中</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelCancel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>35</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px &quot;微软雅黑&quot;;
color:rgba(153,153,153,1);
background-color: rgba(255, 255, 255, 0);
border:0px;</string>
     </property>
     <property name="text">
      <string>取消审核</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
